tools/src/bin/output-file.rs

```diff
@@ -136,6 +136,10 @@ fn main() {
                     title: "Log".to_owned(),
                     link: format!("https://hg.mozilla.org/mozilla-central/log/tip/{}", path),
                     update_link_lineno: false,
+                 }, PanelItem {
+                    title: "Raw".to_owned(),
+                    link: format!("https://hg.mozilla.org/mozilla-central/raw-file/{}/{}", oid, path),
```

Comment:
The oid here is a git revision. You might be able to link to the raw file on github (in the gecko-dev repo). Otherwise, you could link to tip (as the "Log" link does) or figure out a way to convert the Hg revision to a Git revision (which is possible to do with an HTTP API).